### PR TITLE
Added Selu to TF raw ops

### LINKED ISSUE #21271: Selu

### PR DESCRIPTION
**Pull request by mayfieldmobster**
Close #21271

### CODE REVIEW

**Comment by mayfieldmobster** [comment]
spotted a bug

**Comment by KevinUli** [comment]
Hi @mayfieldmobster ,
Thanks for the PR. The implementation looks good to me. There are just slight changes that needs to be made that I have pointed out below. Feel free to re-request a review after the change!

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py [hunk 1]
```diff
@@ -4345,3 +4345,32 @@ def test_tensorflow_Svd(
...
+    frontend,
+    test_flags,
+    fn_tree,
+    backend_fw,
+    on_device,
+):
+    dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        backend_to_test=backend_fw,
```
Comment (KevinUli) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py:4370]: `test_frontend_function` does not seem to have `backend_to_test` kwarg.

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py [hunk 2]
```diff
@@ -4345,3 +4345,32 @@ def test_tensorflow_Svd(
...
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
```
Comment (KevinUli) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py:4374]: You can add `rtol=1e-2` and `atol=1e-2` for small errors in rounding

**Comment by KevinUli** [comment]
Hi @mayfieldmobster ,
Thanks for the PR. The implementation looks good to me. Only 1 minor issue to remove failing test due to slight differences in rounding. After the fix, feel free to re-request a review and I will have it merged to the Ivy codebase!

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py (hunk 2 above)
Comment (KevinUli) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py:4374]: You can add `rtol=1e-2` and `atol=1e-2` to remove failing checks due to small rounding errors.

File: ivy/functional/frontends/tensorflow/raw_ops.py [hunk 3]
```diff
@@ -957,3 +957,15 @@ def Svd(*, input, full_matrices=False, compute_uv=True, name=None):
...
+
+
+@to_ivy_arrays_and_back
+@with_supported_dtypes(
+    {"2.13.0 and below": ("float16", "bfloat16", "float32", "float64")}, "tensorflow"
+)
+def Selu(*, features, name="Selu"):
+    alpha = 1.6732632423543772848170429916717
+    scale = 1.0507009873554804934193349852946
+    return scale * (
```
Comment (juliagsy) [inline ivy/functional/frontends/tensorflow/raw_ops.py:969]: we have a `ivy.selu` function, is there any reason we're doing this compositionally? thanks!

**Comment by juliagsy** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**